lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'dish.rb'
+class Menu
+  attr_reader :dishes
+
+  def initialize(dishes:)
+    @dishes = [dishes]
```

Comment:
This is a bit strange. I would expect `dishes:` to be a list of dishes, but you are initialising `@dishes = [dishes]` - are you putting one array inside another array?